content/en-us/luau/metatables.md

```diff
@@ -323,13 +323,13 @@ When playing with metatables, you may run into some problems. What happens if yo
 
 ```lua
 local t = setmetatable({}, {
-    __index = function(self, i)
-    	self[i] = i * 10 -- just as an example
-    	return self[i]
-    end,
-    __newindex = function(self, i, v)
-    	--don't do anything because we don't want you to set values to the table the normal way
-    end
+	__index = function(self, i)
+		self[i] = i * 10 -- just as an example
+		return self[i]
+	end,
+	__newindex = function(self, i, v)
+		--don't do anything because we don't want you to set values to the table the normal way
+	end,
```

Comment:
```suggestion
	__index = function(self, i)
		self[i] = i * 10  -- Just as an example
		return self[i]
	end,
	__newindex = function(self, i, v)
		-- Don't do anything because we don't want you to set values to the table the normal way
	end,
```